point_cloud_topic: "/camera/depth/points"
filters:
  pass_limits: [-0.2, 0.9, 1.0, 1.0, 0.0, 1.5] # passthrough filter limits: [x_min, x_max, y_min, y_max, z_min, z_max]
  use_voxelgrid_filter: 0 # 0: don't use, 1: use
  leaf_size: 0.01 # voxelgrid filter leaf size
